src/librustc_typeck/error_codes.rs

```diff
@@ -3793,6 +3793,28 @@ details.
 [issue #33685]: https://github.com/rust-lang/rust/issues/33685
 "##,
 
+E0592: r##"
+This error occurs when you defined methods or associated functions with same
+name.
+
+For example, in the following code:
```

Comment:
Please replace this sentence with "Erroneous code example:".

Reply:
Thank you, I replaced with following wording.